src/Sarif.Viewer.VisualStudio.Core/ErrorList/ErrorListService.cs

```diff
@@ -698,6 +703,61 @@ private int WriteRunToErrorList(Run run, string logFilePath, SarifLog sarifLog,
                 this.ShowFilteredSuppressionStateColumn();
             }
 
+            if (skipRemapping == false)
+            {
+                IEnumerable<string> relativeFilePaths = dataCache.SarifErrors.Select(x => x.FileName);
+                IEnumerable<string> uriBaseIds = dataCache.SarifErrors.Select(x => x.SarifResult.Locations?.FirstOrDefault()?.PhysicalLocation?.ArtifactLocation?.UriBaseId);
+
+                // now we need to map from relative file path to absolute.
+                string workingDirectory = dataCache.SarifErrors.FirstOrDefault().WorkingDirectory;
```

Comment:
>workingDirectory [](http://example.com/codeflow?start=23&length=16)

Is there a chance this comes back null? If so, what happens subsequently?

Reply:
added a dataCache.SarifErrors.Any() check in the wrapping block